client/src/components/productReviews/productReviews.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import fetchReviews from '../../actions/fetchReviews';
+
+import Ratings from './ratings.jsx';
+import ReviewsList from './reviewsList.jsx';
+import ReviewButtons from './reviewButtons.jsx';
+import CSS from './productReviews.module.css';
+
+class ProductReviews extends React.Component {
+  componentDidMount() {
+    this.props.handleFetchReviews();
+  }
+
+  render() {
+    return (
+      <>
```

Comment:
what are these carats for?

Reply:
These are [React Fragments](https://reactjs.org/docs/fragments.html),  it can group a few elements together. Sometimes it's handy and I use them to replace `<div>`